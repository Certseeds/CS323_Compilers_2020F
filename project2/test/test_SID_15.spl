int test() {
    int a[114][514][1919][810], b[514][1919][810], c[1919][810], d[810], e;
    int x[810][1919][514][114], y[810][1919][514];
    a[0] = b;
    a[0][114] = c;
    a[0][114][514] = d;
    a[0][114][514][1919] = e;
    b[0] = c;
    c[0] = d;
    d[0] = e;
    x[0] = y;
    y[0] = e;
    y[0][0] = d;
    return a[e][e][e][e];
}


