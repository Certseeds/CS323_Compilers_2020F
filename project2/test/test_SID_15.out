Error type 5 at Line 11: unmatching types on both sides of assignment
Error type 5 at Line 12: unmatching types on both sides of assignment
Error type 5 at Line 13: unmatching types on both sides of assignment
